packages/3id-did-resolver/package.json

```diff
@@ -32,23 +32,20 @@
     "cids": "1.0.2",
     "cross-fetch": "^3.0.6",
     "lru_map": "^0.4.1",
-    "uint8arrays": "^1.1.0"
+    "uint8arrays": "^2.0.5"
   },
   "devDependencies": {
     "@babel/core": "^7.9.0",
     "@babel/preset-env": "^7.8.4",
     "@babel/preset-typescript": "^7.9.0",
-    "@types/bs58": "^4.0.1",
-    "@types/events": "^3.0.0",
     "@types/node": "^13.13.15",
     "@typescript-eslint/eslint-plugin": "^4.6.0",
     "@typescript-eslint/parser": "^4.6.0",
-    "babel-jest": "^25.1.0",
+    "babel-jest": "^26.6.3",
```

Comment:
At some point it might make more sense to switch to `ts-jest` so that we don't need all of these babel deps.

Reply:
Interesting point. To date I thought that was an explicit decision to jump onto Babel bandwagon. The most obvious reason I saw is speed - Babel avoids type checking.

Personally, prefer ts-jest. Less clutter in the dependency list.

Reply:
I think when I initially set up this repo I was not aware of ts-jest. How big is the speed difference?

Reply:
Have seen anecdotal evidence only like https://iamturns.com/typescript-babel/

Fanboy-ish "it's fast".

In my experience, it's like 1.2-1.5 times slower. Not good, but not critical, and it really complains on broken tests wrt types.

Reply:
So let's keep babel unless we have an explicit reason to change?
cc @zachferland if you have any thoughts here.